lib/amqp/gen/consumer.ex

```diff
@@ -197,8 +198,19 @@ defmodule Amqpx.Gen.Consumer do
     e in _ ->
       Logger.error(inspect(e))
 
+      is_message_to_reject = function_exported?(handler_module, :handle_message_rejection, 1) && redelivered
+
       Task.start(fn ->
         :timer.sleep(backoff)
+
+        case is_message_to_reject do
```

Comment:
In questo caso, dal momento che si tratta di un `case` con solo due rami, può essere utilizzato un `if`